pkg/cmd/opts/common.go

```diff
@@ -1142,3 +1142,16 @@ func (o *CommonOptions) IsFlagExplicitlySet(flagName string) bool {
 	o.Cmd.Flags().Visit(explicitlySetFunc)
 	return explicit
 }
+
+// GetClusterName returns the current cluster name
+func (o *CommonOptions) GetClusterName() (string, error) {
```

Comment:
We have already a helper function for this. I would be great to reuse it.
https://github.com/jenkins-x/jx/blob/442ddfc3176f50ef463faecb0152397dc861df38/pkg/kube/cluster/cluster.go#L14

Reply:
thanks! figure there must be but couldn't find it - fixed!